Storage/SQL/BrowserSchema.swift

```diff
@@ -1301,7 +1301,7 @@ open class BrowserSchema: Schema {
         return true
     }
 
-    func drop(_ db: SQLiteDBConnection) -> Bool {
+    public func drop(_ db: SQLiteDBConnection) -> Bool {
```

Comment:
Why does this need to be public? Why not just `create` and `update`?

Reply:
Because its part of the `Schema` protocol.

Reply:
Does it need to be part of the `Schema` protocol?

Reply:
Yes. It is called from `ConcreteSQLiteDBConnection` just the same as `create` and `update`.